src/core/best-practices.md

```diff
@@ -111,3 +111,47 @@ dispersion of 1 degree could classify two separate fixations instead of one. Set
 dispersion too high might group fixations together that should have been considered separate fixations. 
 Ensure the maximum dispersion accommodates your viewing task. 
 
+## Sychronization
+Pupil Core is often used concurrently with third-party devices and software (e.g. physiological sensors, 
+motion capture, stimuli presentation). In order to correlate the data between these, temporal alignment is of great importance.
+
+How you go about synchronizing Pupil Core with other devices or software will depend on the setup, but there are three 
+common approaches (steps 2–3 leverage our [Network API](/developer/core/network-api)):
+
+::: tip
+<v-icon large color="info">info_outline</v-icon>
+Read about Pupil Core's [timing definitions here](/core/terminology/#timing).
+:::
+
+::: tip
+<v-icon large color="info">info_outline</v-icon>
+When using multiple Pupil Core devices (running on the same or multiple machines), be sure to turn-on 
```

Comment:
```suggestion
When using multiple Pupil Core devices (running on the same or multiple machines), be sure to turn on
```
Isn't `turn-on` a noun? 🙃

Reply:
Inappropriate use of language. I apologise😬